crates/core/tedge/src/cli/certificate/cli.rs

```diff
@@ -37,10 +35,6 @@ pub enum TEdgeCertCli {
         #[clap(long = "device-id", global = true)]
         id: Option<String>,
 
-        /// Path where a Certificate signing request will be stored
-        #[clap(long = "output-path", global = true, value_hint = ValueHint::FilePath)]
-        output_path: Option<Utf8PathBuf>,
```

Comment:
Why `--output_path` option is removed?

Reply:
Oops, I noticed when preparing tests that it actually should be here 👀. I reversed this change.